pages/ProfileView.neu.page.ts

```diff
@@ -0,0 +1,111 @@
+  }
+
+  public async changePassword(username: string, password: string): Promise<string> {
+    await this.page.getByTestId('open-change-password-dialog-button').click();
+    await this.page.getByTestId('change-password-button').click();
+
+    await expect(this.page.locator('#kc-attempted-username')).toHaveText(username);
+    await expect(this.page.getByTestId('login-prompt-text')).toHaveText('Bitte geben Sie Ihr aktuelles Passwort ein.');
+    await this.page.getByTestId('password-input').fill(password);
+    await this.page.getByTestId('login-button').click();
+
+    const newPassword: string = await this.loginViewPage.updatePassword();
+    await this.page.getByTestId('close-password-changed-dialog-button').click();
+
+    await expect(this.page.getByTestId('profile-headline')).toHaveText('Mein Profil');
+    return newPassword;
+  }
+
+  /* assertions */
+  public async checkPersoenlicheDaten(persoenlicheDaten: PersoenlicheDaten): Promise<void> {
+    await expect(this.page.getByTestId('layout-card-headline-persoenliche-daten')).toHaveText('Persönliche Daten');
+    await expect(this.page.getByTestId('fullname-label')).toHaveText('Vor- und Nachname:');
+    await expect(this.page.getByTestId('fullname-value')).toHaveText(persoenlicheDaten.vorname + ' ' + persoenlicheDaten.nachname);
+    await expect(this.page.getByTestId('username-label')).toHaveText('Benutzername:');
+    await expect(this.page.getByTestId('username-value')).toHaveText(persoenlicheDaten.username);
+
+    if (persoenlicheDaten.rollenart === 'LEHR') {
+      await expect(this.page.getByTestId('kopersnummer-label')).toHaveText('KoPers.-Nr. :');
+      await expect(this.page.getByTestId('kopersnummer-value')).toBeVisible();
+    } else {
+      await expect(this.page.getByTestId('kopersnummer-label')).toBeHidden();
+      await expect(this.page.getByTestId('kopersnummer-value')).toBeHidden();
+    }
+    await expect(this.page.getByTestId('info-icon')).toBeVisible();
+  }
+
+  // for each zuordnung, check if card is visible and contains the correct data
+  public async checkZuordnungen(zuordnungen: Zuordnung[]): Promise<void> {
+    for (let i: number = 0; i < zuordnungen.length; i++) {
+      const zuordnung: Zuordnung = zuordnungen[i];
+
+      if (zuordnungen.length === 1) {
+        await expect(this.page.getByTestId(`zuordnung-card-${i}-headline`)).toHaveText('Schulzuordnung');
+      } else {
+        await expect(this.page.getByTestId(`zuordnung-card-${i}-headline`)).toHaveText('Schulzuordnung ' + (i + 1));
+      }
+
+      await expect(this.page.getByTestId(`schule-label-${i}`)).toHaveText('Schule:');
+      await expect(this.page.getByTestId(`schule-value-${i}`)).toHaveText(zuordnung.organisationsname);
+      await expect(this.page.getByTestId(`rolle-label-${i}`)).toHaveText('Rolle:');
+      await expect(this.page.getByTestId(`rolle-value-${i}`)).toHaveText(zuordnung.rollenname);
```

Comment:
```suggestion
  await Promise.all([
    expect(this.page.getByTestId(`zuordnung-card-${i}-headline`)).toHaveText(zuordnungen.length === 1 ? 'Schulzuordnung' : `Schulzuordnung ${i + 1}`),
    expect(this.page.getByTestId(`schule-label-${i}`)).toHaveText('Schule:'),
    expect(this.page.getByTestId(`schule-value-${i}`)).toHaveText(zuordnung.organisationsname),
    expect(this.page.getByTestId(`rolle-label-${i}`)).toHaveText('Rolle:'),
    expect(this.page.getByTestId(`rolle-value-${i}`)).toHaveText(zuordnung.rollenname),
  ]);
```
Could parallelize those checks for performance maybe?